### ISSUE TITLE
[Bug] Lyrics which aren't synced have no option to manually scroll through them

### ISSUE THREAD

**Issue by lukemattle** [gloaysa/whatsplaying#5]
Lyrics which are not synced to the line but still exist are stuck at their first line in the interface. Ideally, you should be able to manually scroll through them.
Otherwise, this is a **great** little program (and as far as I can tell is the only way to persistently show lyrics for songs that have them)!

**Comment by gloaysa** [comment]
Yes! I have noticed it once (don’t have many lyrics not timed, so I need to find the song again to test this one out).
Plex api actually tells you if the lyric is timed or not. Didn’t implemented it out of laziness, tbh 😛 

To the backlog!
I won’t be able to work on any of this until the weekend. I’d recommend you to subscribe to the releases if you are interested to get notified when I release new stuff. 

I also have pending adding an update notification to the app. 

Glad you like it! Is so cool when you make something and others find it useful :)
